rust/otel-arrow-rust/Cargo.toml

```diff
@@ -22,7 +22,7 @@ trace = []
 derive = []
 
 [dependencies]
-arrow = "55"
+arrow = {version = "55", features = ["prettyprint"] }
```

Comment:
Curious -- is it possible to use a crate and have a feature be a dev-dependency? I noticed a call to arrow::util::prettyprint and I can see it being useful in tests and examples.